src/server/origin/summary/controller.js

```diff
@@ -0,0 +1,45 @@
+const indexView = 'origin/summary/index.njk'
+export const pageTitle =
+  'Check your answers before you continue your application'
+export const pageHeading =
+  'Check your answers before you continue your application'
+
+/**
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const originSummaryGetController = {
+  handler(req, h) {
+    const origin = req.yar.get('origin') ?? {}
+
+    const originOnOffFarm = origin?.onOffFarm
+    let enteredOnOffFarm = ''
+
+    if (originOnOffFarm === 'on') {
+      enteredOnOffFarm = 'On to the farm or premises'
+    } else if (originOnOffFarm === 'off') {
+      enteredOnOffFarm = 'Off the farm or premises'
+    } else {
+      enteredOnOffFarm = ''
+    }
```

Comment:
At some stage, I'm keen to move this into the session state so it can be looked up dynamically.

we'll be re-using this in the overall summary, and then again when we send that data for review by the admin staff, so it's important to have a common place to access it from.

I think this is probably a target for a later refactoring though?